dataflow/src/main/java/org/checkerframework/dataflow/analysis/Analysis.java

```diff
@@ -132,23 +140,46 @@ public void setCurrentTree(Tree currentTree) {
     /**
      * Construct an object that can perform a org.checkerframework.dataflow analysis over a control
      * flow graph. The transfer function is set later using {@code setTransferFunction}.
+     *
+     * @deprecated will be removed in next version
```

Comment:
This comment isn't helpful. See https://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#@deprecated for how to write a deprecated comment.  Please update all `@deprecated` comments.